content/source/docs/glossary.html.md

```diff
@@ -0,0 +1,560 @@
+Terraform Enterprise (TFE) Glossary
+
+## API
+
+"Application Programming Interface". Any interface designed to allow programatic manipulation of some kind of software system. For most software developers today, the most common kinds of APIs are based on HTTP requests.
+
+Terraform relies on cloud service provider APIs to manage resources; each service's Terraform provider is responsible for mapping Terraform's resource model to the series of actual API calls necessary to create, check, modify, or delete a real infrastructure resource in that cloud service.
+
+Terraform Enterprise also offers its own API, for managing resources like team membership, policies, and workspaces. That API, in turn, is used by the `tfe` Terraform provider, so you can use Terraform to manage the system that runs Terraform for you.
+
+- [TFE docs: API](/docs/enterprise/api/index.html)
```

Comment:
I'm wondering if the bulleted links should be prefixed with `*See also:*` to give them more context. But I'm also feeling like maybe that's not necessary.

Reply:
I initially did that, and then rethought it. Could still go either way!